.github/workflows/deploy-branch.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+set -eux
```

Comment:
```suggestion
set -euxo pipefail
```
You probably want to be told when your curl call fails on line 20.